grails-app/services/org/pih/warehouse/dashboard/IndicatorDataService.groovy

```diff
@@ -102,7 +107,8 @@ class IndicatorDataService {
         return graphData
     }
 
-    GraphData getFillRate(Location location, def destination, def params) {
+    @Cacheable(value = "dashboardCache", key = { "getFillRate-${location.id}${destination?.id ?: ''}${params.querySize}${params.listFiltersSelected}${params.value}" })
```

Comment:
For now, we can't use individual cache per indicator, because `@CacheEvict` cannot have an array of cache names as an argument, so we can't clear more than one cache using it. Because of that, we won't be able to refresh caches for all indicators.